x/dispensation/keeper/querier_test.go

```diff
@@ -44,24 +44,24 @@ func TestQueryRecordsName(t *testing.T) {
 		Path: "",
 		Data: []byte{},
 	}
-	qp, errRes := app.Codec().MarshalJSON(quereyRecName)
+	qp, errRes := sifapp.AppCodec().MarshalJSON(&quereyRecName)
```

Comment:
@marbar3778 the unmarshalling from here is failing now that we've implemented status as an enum, regardless of enum value passed here, can you check out this issue?

`cannot unmarshal string into Go value of type types.ClaimStatus: failed to unmarshal JSON bytes`